### PR TITLE
update changelog with 3.7 and 3.8

### PR DESCRIPTION
**Pull request by michalb-sonar**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Contributors to this file, please follow the guidelines on [keepachangelog.com](
 
 For reference, the possible headings are:
 
-* **New Feature** for new features.
+* **New Features** for New Featuress.
```
Comment (duncanp-sonar) [inline CHANGELOG.md]: Typo - double "s" at end of the line.

Comment (michalb-sonar) [reply CHANGELOG.md]: Clbuttic mistake, thanks for spotting!

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,110 +6,134 @@ Contributors to this file, please follow the guidelines on [keepachangelog.com](
...
+
+
+## [3.7]
+
+### Bugs
+* [#395](https://github.com/SonarSource/sonarlint-visualstudio/issues/395) - Binding the solution to SonarQube before all projects have finished to load result in weird error
+* [#413](https://github.com/SonarSource/sonarlint-visualstudio/issues/413) - Opening a solution with duplicate solution folder names crashes VS
+
+### Improvements
+* [#347](https://github.com/SonarSource/sonarlint-visualstudio/issues/347) - "Connect" option is not available unless a solution is open
```
Comment (Evangelink) [inline CHANGELOG.md]: Let's rename this ticket to actually state the solution: "Connect option should be available only after a solution is loaded".

Comment (michalb-sonar) [reply CHANGELOG.md]: I don't think we should put the solution in ticket names. That would mean a lot of renaming after tickets are addressed.

But agree for renaming in this instance. The name is a bit confusing, your suggestion is much better.

**Comment by Evangelink** [comment]
I was more suggesting to pay attention to ticket names so that they always have a clear meaning. And along those lines I have realized it is often more obvious to state the actual problem for a bug and to state the actual improvement for an improvement (rather than just describing the problem).

**Pull request merged**